modules/howtos/pages/encrypting-using-sdk.adoc

```diff
@@ -1,14 +1,152 @@
-= Field Level Encryption from the SDK
+= Field Level Encryption from the Go SDK
 :page-topic-type: howto
 :page-edition: Enterprise Edition
-:page-status: Developer Preview
-// // :page-aliases: ROOT:encrypting-using-sdk.adoc
+// :page-aliases: ROOT:encrypting-using-sdk.adoc

```

Comment:
Think we need to uncomment this

Reply:
Note - not until https://github.com/couchbase/docs-sdk-common/blob/release/6.6/modules/pages/partials/encryption.adoc (which has the same alias) is updated.
Otherwise Antora will not build.